Marlin/src/lcd/e3v2/common/encoder.cpp

```diff
@@ -175,7 +174,7 @@ EncoderState encoderReceiveAnalyze() {
   //  luminance: brightness (0~0xFF)
   //  change_Time: gradient time (ms)
   void LED_GraduallyControl(const uint8_t RGB_Scale, const uint8_t luminance, const uint16_t change_Interval) {
-    struct { uint8_t g, r, b; } led_data[LED_NUM];
+    struct { uint8_t r, g, b; } led_data[LED_NUM];
```

Comment:
The standard for most LEDs is G-R-B so are you certain the standard model E3V2 is R-G-B ?? All other code referring to LEDs in this file presumes G-R-B.